homeassistant/components/opi_gpio.py

```diff
@@ -0,0 +1,68 @@
+"""
+Support for controlling GPIO pins of a Orange Pi.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/opi_gpio/
+"""
+# pylint: disable=import-error
+import logging
+
+from homeassistant.const import (
+    EVENT_HOMEASSISTANT_START, EVENT_HOMEASSISTANT_STOP)
+
+REQUIREMENTS = ['git+git://github.com/Toshik/orangepi_PC_gpio_pyH3.git@master#pyA20==0.2.1']
```

Comment:
Original fork messed up pin numbers. Also I have plans to implement interrupts in my fork.